### ISSUE TITLE
GitHub Infrastructure Page Creation Tracker

### ISSUE THREAD

**Issue by patrick-5546** [UBCSailbot/docs#24]
### Purpose
Document GitHub infrastructure

#### Page Tree
```yml
nav:
  ...
  - Reference:
    ...
    - Infrastructure:
      - GitHub:
        - Development Workflow:
          ...
        - Features:
          ...
        - Repositories:
          ...
      ...
```

#### Tasks
- [ ] https://github.com/UBCSailbot/docs/issues/103 -- **must be done before next item in list**
- [ ] https://github.com/UBCSailbot/docs/issues/104 -- **must be done before next item in list**
- [ ] https://github.com/UBCSailbot/docs/issues/99
- [ ] https://github.com/UBCSailbot/docs/issues/100
- [ ] https://github.com/UBCSailbot/docs/issues/101

### Relevant Resources
- Google Docs
    -  GitHub Organization
- Confluence
    - Infrastructure meeting notes

**Comment by patrick-5546** [comment]
Update: put on hold, as it is still in active development. Write once we have started developing actual new project codebase

**Comment by patrick-5546** [comment]
Split into multiple issues

**Comment by DFriend01** [comment]
Converted issue to task list

**Comment by patrick-5546** [comment]
Maybe we should do this in the `Current Project/Sailbot Workspace` section for test_definitions.yml and reusable workflows, then `Current Project/<repo>` for any CI unique to a repository

Yea I think we should focus on documenting our own infrastructure rather than creating a general reference for something that changes all the time
